app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/Answer.kt

```diff
@@ -15,7 +15,7 @@ class Answer {
 
     @SerializedName("data")
     @Expose
-    var data: List<Map<String, Any>>? = null
+    var data: List<Map<String, Any>> = ArrayList()
```

Comment:
Object

Reply:
@iamareebjamal android studio is not allowing using Object

Reply:
Create a new class is what I mean